docs/pages/concepts/architecture/broker.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Broker
+permalink: concepts/architecture/broker
+type: concepts
+redirect_from: architecture/broker
+abstract: "Meshery broker component fascilitates data streaming between kubernetes cluster components and outside world."
+language: en
+list: include
+---
+
+Broker is a custom kubernetes controller that empowers data streaming across different components of meshery, which are either inside or outside of the kubernetes cluster.
+
+### Broker FAQs
```

Comment:
- How many Brokers can run? 
- What does an HA configuration look like? 
- What stateful characteristics does the Broker have?
- How do I know if the Broker is working? How do I troubleshoot the Broker?

Reply:
Done done..